lib/matplotlib/figure.py

```diff
@@ -1440,22 +1443,22 @@ def legend(self, *args, **kwargs):
             If shadow is activated and framealpha is ``None`` the
             default value is being ignored.
 
-        facecolor : None or "inherit" or a color spec
+        facecolor : None or 'inherit' or a color spec
             Control the legend's background color.
             Default is ``None`` which will take the value from the
             ``legend.facecolor`` :data:`rcParam<matplotlib.rcParams>`.
-            If ``"inherit"``, it will take the ``axes.facecolor``
+            If ``'inherit'``, it will take the ``axes.facecolor``
             :data:`rcParam<matplotlib.rcParams>`.
 
-        edgecolor : None or "inherit" or a color spec
+        edgecolor : None or 'inherit' or a color spec
             Control the legend's background patch edge color.
             Default is ``None`` which will take the value from the
             ``legend.edgecolor`` :data:`rcParam<matplotlib.rcParams>`.
-            If ``"inherit"``, it will take the ``axes.edgecolor``
+            If ``'inherit'``, it will take the ``axes.edgecolor``
             :data:`rcParam<matplotlib.rcParams>`.
 
-        mode : {"expand", None}
-            If `mode` is set to ``"expand"`` the legend will be horizontally
+        mode : {'expand', None}
+            If `mode` is set to ``'expand'`` the legend will be horizontally
```

Comment:
Please revert all changes up to here.  I think what follows looks good.

Reply:
This one and similar ones reverted up to here. There were however some plain mistakes above, which lead to broken Sphinx rendering. I did not revert these.